src/Features/Core/Portable/ExtractMethod/MethodExtractor.CodeGenerator.cs

```diff
@@ -26,6 +26,10 @@ internal abstract partial class AbstractExtractMethodService<
 {
     internal abstract partial class MethodExtractor
     {
+        public static readonly SyntaxAnnotation MethodNameAnnotation = new();
+        public static readonly SyntaxAnnotation MethodDefinitionAnnotation = new();
+        public static readonly SyntaxAnnotation CallSiteAnnotation = new();
```

Comment:
made these static readonlies.  tehre are always 3 per individual extract method op.  no need to make fresh instances which then have to be passed around.